docs/components/concepts/access-control/mapping-rules.md

```diff
@@ -0,0 +1,72 @@
+---
+id: mapping-rules
+title: "Mapping rules"
+sidebar_label: "Mapping rules"
+description: "Map authentication entities from your Identity provider to Camunda-specific entities using mapping rules."
+---
+
+Mapping rules are used to dynamically manage access control in Camunda by [connecting your Identity
+Provider](connect-to-identity-provider.md) and assigning claims from a JWT access token to [Orchestration
+Cluster Identity](../../identity/identity-introduction.md) entities.
+
+## Support of mapping rules in SaaS and Self-Managed
+
+In **Camunda 8 SaaS**, you cannot currently configure mapping rules. They are automatically resolved based on your organization's setup.
+
+In **Camunda 8 Self-Managed**, you may need to configure mapping rules in two components:
+
+- **Orchestration Cluster Identity**: These mapping rules manage authorizations within an [orchestration cluster](../../orchestration-cluster.md). They can be used to assign users to [user groups](../../identity/group.md) and [roles](../../identity/role.md), grant [authorizations](../../identity/authorization.md), and
```

Comment:
First sentence can be missunderstood since tenants are no authorizations. Second sentence clarifies it, but perhaps can be formulated a bit clearer